source/Application/Model/Article.php

```diff
@@ -488,6 +488,10 @@ public function __construct($aParams = null)
                 $this->$sParam = $mValue;
             }
         }
+        
+        // move this and the following dependencies to the constructor and do not use getters inside of methods
+        $this->config = $this->getConfig();
```

Comment:
ofc much nicer would be to do this in Base and Multilang classes, so children just access them via ``$this->...``

i guess you got the point. and not BC breaks with this change!